packages/core/src/execution-engine/node-execution-context/webhook-context.ts

```diff
@@ -160,7 +177,7 @@ export class WebhookContext extends NodeExecutionContext implements IWebhookFunc
 		return await getInputConnectionData.call(
 			this,
 			this.workflow,
-			runExecutionData,
+			this.runExecutionData!,
```

Comment:
Can you keep this logic and the fallback above as it was before? Otherwise I'm afraid that if the implementation of the constructor changes later, this `!` will hide valid type errors.

Reply:
Sure! Done.